Style error in pkg/client

```
==> Running golint...

pkg/client/deployments.go:103:1: comment on exported method Client.DescribeDeployment should be of the form "DescribeDeployment ..."
```

--- a/pkg/client/client.go
+++ b/pkg/client/client.go
@@ -226,27 +226,28 @@ func (r *Response) HTTPError() error {
 
 	defer r.Body.Close()
 	body, err := ioutil.ReadAll(r.Body)
+	if err != nil {
+		return err
+	}
 
 	if contentType == "application/json" {
 		httpErr := httputil.Error{}
-		json.Unmarshal(body, &httpErr)
+		if err := json.Unmarshal(body, &httpErr); err != nil {
+			return err
+		}
 		return &HTTPError{
 			StatusCode: r.StatusCode,
 			Message:    httpErr.Msg,
 			URL:        r.Request.URL,
 		}
-	} else {
-		if err != nil {
-			return err
-		}
-		return &HTTPError{
-			StatusCode: r.StatusCode,
-			Message:    string(body),
-			URL:        r.Request.URL,
-		}
 	}
 
-	return nil
+	return &HTTPError{
+		StatusCode: r.StatusCode,
+		Message:    string(body),
+		URL:        r.Request.URL,
+	}
+
 }
 
 // HTTPError is an error caused by an unexpected HTTP status code.

--- a/pkg/client/deployments.go
+++ b/pkg/client/deployments.go
@@ -100,7 +100,7 @@ func (c *Client) DeleteDeployment(name string) (*common.Deployment, error) {
 	return deployment, err
 }
 
-// DescribDeployment describes the kubernetes resources of the supplied deployment
+// DescribeDeployment describes the kubernetes resources of the supplied deployment
 func (c *Client) DescribeDeployment(name string) (*common.Deployment, error) {
 	var deployment *common.Deployment
 	//TODO: implement

